src/main/scala/is/hail/expr/AnnotationImpex.scala

```diff
@@ -227,26 +227,26 @@ object JSONAnnotationImpex extends AnnotationImpex[Type, JValue] {
 
       val vfs = f()
 
-      vfs(0).flatMap { chr =>
-        vfs(1).flatMap { pos =>
-          vfs(2).flatMap { ref =>
-            vfs(3).map { alt =>
-              Variant(chr.asInstanceOf[String],
-                pos.asInstanceOf[Int],
-                ref.asInstanceOf[String],
-                alt.asInstanceOf[IndexedSeq[String]].toArray)
-            }
-          }
-        }
-      }
+      val chr = vfs(0)
+      val pos = vfs(1)
+      val ref = vfs(2)
+      val alt = vfs(3)
+
+      if (chr != null && pos != null && ref != null && alt != null)
+        Variant(chr.asInstanceOf[String],
+          pos.asInstanceOf[Int],
+          ref.asInstanceOf[String],
+          alt.asInstanceOf[IndexedSeq[String]].toArray)
```

Comment:
I wish there were a way to get the array back out of a WrappedArray without copying.  There's not, is there?